CONTRIBUTING.md

```diff
@@ -123,6 +123,26 @@ This will install tern in development mode on windows and mac.
 ### Testing you changes
 After you make the changes just run `tox`. This will run a set tests and inform you if anything is wrong.
 
+### Setting up a development environment for use with extensions
+Tern will be able to find the absolute path of the command line tool which will be used as an extension if it is available on the system. For instance, to make Tern work with Scancode, you can install Scancode in your development environment such that it is available when you run `which scancode`.
```

Comment:
Do we want to tell them how they install ScanCode?

Reply:
How about a sentence about using tern with extensions with the link back to the specific place in the README?